free-1.md

```diff
@@ -1,68 +1,170 @@
+
+
+
 ### Why a language?
+
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
 
+
+If existing map makers count as DSLs, then these tools have already proven to address this type of need.
+
+
+
 ### Why you?
+
 _What excites you about this idea? How did you come up with it?_
 
 
+
+Words cannot express my interest in game design.
+
+Map generation is one of the basics of game dev tools, and free-to-use tools can always be improved on.
+
+
+
 ### Domain
+
 _Describe the project's domain in five words._
 
 
+
+Tile map editing and generation
+
+
+
 ### Interface (syntax)
+
 _How might the user interact with the language? What does programming look 
+
 like? Why is this the right way to interact with the problem domain?_ 
 
 
+
+A "program" would be a breakdown of a map, with tile patterns specified for each area, potentially how the tiling should be aligned, and with any objects to be layered over this.
+
+
+
 ### Operation (semantics)
+
 _What might happen when a program runs? How does a program interact with the
+
 user? What kinds of errors might occur, and how might they be communicated to
+
 the user?_
 
 
+
+When the program runs, it would generate any created tiles, then repeat the patterns and use the map specifications to create each piece, then stitch them together and output an image.
+
+If any area is left blank, it should probably color it in with an obnoxiously bright color so the user knows to cover that area.
+
+
+
 ### Expressiveness
+
 _What should be easy to do in this language? What should be possible, but
+
 difficult? What should be impossible or very difficult?_
 
 
+
+This all depends on constraints. Resizing things may prove difficult.
+
+
+
 ### Related work
+
 _Are there any other DSLs in this domain? If not, describe how you know there
+
 aren't and conjecture why not. If so, describe them and provide links. How well 
+
 do they address the need? Are there any particularly admirable qualities of the
+
 language? Are there parts of the language you think could be improved?_
 
 
+
+There are loads of RPG mapmakers. Most of them have pre-defined tile sizes, but with enough searching it may be possible to find one that has this behavior; however, overall they give game developers enough tools to succeed. 
+
+
+
 ## The Project
+
 This section examines whether the idea makes for a good CS 111 project.
 
 
+
+This is probably too big an idea to actually pull off, but parts of it might get done. I'm also not sure if it's enough of a simulation language to be considered a good DSL.
```

Comment:
What is a simulation language? What would make this project too big to pull off?